pkg/agent/rest.go

```diff
@@ -115,6 +119,27 @@ func FindRendezvouIPAndSSHKeyFromAssetStore(assetDir string) (string, string, er
 	return rendezvousIP, sshKey, nil
 }
 
+// FindAuthTokenFromAssetStore returns the auth token.
+func FindAuthTokenFromAssetStore(assetDir string) (string, error) {
+	authConfigAsset := &gencrypto.AuthConfig{}
+
+	assetStore, err := assetstore.NewStore(assetDir)
```

Comment:
Creating a new asset store results in quite a bit of noise in the debug logs I think. In future it would be nice to refactor this so we only use the one asset store for both this and getting the Rendezvous IP.